apps/src/redux/instructions.js

```diff
@@ -307,7 +310,7 @@ export const determineInstructionsConstants = config => {
   let shouldShowOverlay = hasInstructionsToShow &&
       (config.level.instructionsImportant || config.levelPosition === 1 || !overlaySeen);
   if (shouldShowOverlay) {
-    localStorage.setItem(LOCALSTORAGE_OVERLAY_SEEN_FLAG, true);
+    trySetLocalStorage(LOCALSTORAGE_OVERLAY_SEEN_FLAG, true);
```

Comment:
note that this instance is the one that was actually causing the page load error; the others would cause errors only on specific actions.